pages/your_tasks/machine_actionability.md

```diff
@@ -24,11 +24,13 @@ faircookbook:
   url: https://w3id.org/faircookbook/FCB081
 ---
 
-## What does machine-readable, machine-actionable or machine-interpretable mean for data and metadata in RDM?
+## What does machine-readable, machine-actionable or machine-interpretable mean?
```

Comment:
the use of _or_ suggests that answer will address one of the terms not all three terms. As I understand, the question here should be What does machine-readable, machine-actionable**, and** machine-interpretable mean?

Reply:
Done!